docs/v1/introduction.mdx

```diff
@@ -1,23 +1,60 @@
 ---
-title: Introduction
-description: "Build your next agent with evals, observability, and replay."
+title: "Introduction"
+description: "Build your next agent with evals, observability, and replays"
+mode: "wide"
 ---
 
-## Get Started
+<Check>[Give us a star](https://github.com/AgentOps-AI/agentops) on GitHub if you find AgentOps helpful (you may be our 1,000th 😊)</Check>
 
-<CardGroup cols={2}>
-  <Card title="Installing AgentOps" icon="wrench" href="./installation">
-    Prepare your system for AgentOps
-  </Card>
-  <Card title="Quick Start" icon="rocket" href="./quickstart">
-    Start using AgentOps with just 3 lines of code
-  </Card>
-</CardGroup>
+# AgentOps is your new terminal
 
-## Integrate with your agent
+<Card title="Terminals suck" icon="trash" iconType="solid" color="#CC0000">
+	- • Lose track of what your agents did in between executions
+	- • Parsing through terminal output searching for LLM completions
+	- • Printing "tool called"
+</Card>
+<Card title="AgentOps doesn't" icon="face-smile-halo" iconType="duotone" color="#00CC00">
```

Comment:
"AgentOps doesn't" is confusing to me. I feel like "AgentOps doesn't suck" is better

Reply:
<img width="882" alt="Screenshot 2024-05-23 at 4 33 55 PM" src="https://github.com/AgentOps-AI/agentops/assets/15098079/8f3ec5f0-2a98-4387-951d-57aaa6ee73fa">

Reply:
I think visually it's pretty clear